src/js/utils/general.js

```diff
@@ -230,3 +232,13 @@ function _makeCenteredTippy(step) {
 
   return tippy(document.body, tippyOptions);
 }
+
+export function normalizePrefix(prefix) {
+  if (typeof prefix !== 'string' || prefix === '') {
+    return '';
+  }
+  if (prefix.charAt(prefix.length - 1) !== '-') {
```

Comment:
I believe we could simplify this method to:

```js
if(!isString(prefix) || prefix === '') {
  return '';
}

return prefix.endsWith('-') ? prefix : `${prefix}-`;
```

Reply:
I've simplified.
but `endsWith` is [not supported by IE11](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith#Browser_compatibility) using additional polifill just for this line seems too much.